### DISCUSSION TITLE
Vorstellung: Umsetzung OpenDTU-OnBattery mit Komponenten aus dem Verteilerbau

### DISCUSSION THREAD

**Discussion by michaelmaximilian** [hoylabs/OpenDTU-OnBattery#1347] [Show and tell]
Hallo liebe OpenDTU-OnBattery Community,

ich bin seit vielen Monaten Mitleser in diesem Projekt. Die vielen Informationen aus den Diskussionen und dem Wiki haben mir sehr geholfen, das Projekt innerhalb kürzester Zeit zum Laufen zu bekommen. Vielen Dank an alle Beteiligten!

Als **Inspiration und weiteres Beispiel für eine mögliche Hardware-Umsetzung** möchte ich einige Details zu meinem Aufbau mit Euch teilen. Da ich in meiner Mietswohnung leider keine Werkstatt habe, habe ich mich entschieden, die Stromverteilung mit Komponenten aus dem Verteilerbau zu realisieren:

![IMG_2506](https://github.com/user-attachments/assets/3b536ead-41f3-44f5-a262-653afe99c3f2)
(im Bild fehlt noch die Abdeckung des Verteilerkastens sowie die PV-Zuleitung)

Die Sicherungen sind von ABB (S200-Serie). Die Sicherungen sind für den Gleichstrombetrieb bis 75V freigegeben (unabhängig von der Einspeiserichtung!). Durch Reihenschaltung von Sicherungskammern kann die zulässige Abschaltspannung weiter erhöht werden. Die Datenblätter enthalten entsprechende Verschaltungsbeispiele. Ich habe mich bei der Sicherungsauswahl bewusst gegen die günstigen China-Sicherungen entschieden, da diese in manchen Tests nicht zuverlässig trennen konnten.

Die Stromverteilung erfolgt über Verteiler der Firma Wago und Phönix Contact (zugelassen bis 63A). Durch die Klemmtechnik, die keine Aderendhülsen erfordert, lässt sich der Verteiler sehr schnell aufbauen.

Mein ESP32 läuft auf dem Board von Lukas. Die Spannungsversorgung leitet sich aus 230V ab (Netzteil von Mean Well für die Hutschiene). Der Stromzähler ist über einen Hichi-Lesekopf angebunden.

**Was würde ich beim nächsten Mal anders machen / sonstige Erkenntnisse:**

- Die 10qmm Verdrahtungskabel sind zu starr und haben große Biegeradien. Ich würde die gesamte Verkabelung mit flexiblen Silikonkabeln machen.
- Victron empfiehlt, den Minuspol des Systems zu erden; das macht in diesem Aufbau keinen Sinn. Der Hoymiles-Wechselrichter erkennt das als "Isolationsfehler" und startet nicht.
- Eine gute WLAN-Versorgung ist die Voraussetzung für eine stabile Regelung; Repeater oder Powerline-Adapter helfen, den Empfang im Keller zu verbessern.
- Wählt den Platz für die einzelnen Komponenten nicht zu klein aus; die Verkabelung benötigt mehr Platz als erwartet.
- Bringt das System Schritt für Schritt zum Laufen (also z.B. erst einmal die Batterie mit dem ESP32 auslesen; dann den Stromzähler dazunehmen usw...). Das vereinfacht die Fehlersuche.
- An sehr guten Sommertagen ist der Speicher bereits nach wenigen Stunden voll und das System regelt ab; für die überschüssige Energie suche ich noch einen Verbraucher im Haus (zusätzlich werde ich Full Solar Passthrough demnächst einmal testen).
- In den letzten fünf Monaten hat sich der HMS-800-2T zwei Mal aufgehängt; dazu gibt es auch entsprechende Fehlertickets von anderen Betroffenen. Hier hilft nur ein Neustart des Systems durch komplettes Runterfahren der DC-Spannung am Wechselrichter. Hier steht noch ein Firmware-Update des Wechselrichters auf meiner ToDo-Liste (in der Hoffnung, dass der sporadische Fehler damit evtl. behoben wird).
- Achtet auf etwas Abstand zwischen dem Wechselrichter und anderen Komponenten; in meinem Beispiel hängt der Wechselrichter frei in der Luft, der Abstand beträgt 2cm zur Mauer; unter Vollast wird der Wechselrichter maximal 50°C warm (weshalb ich mich gegen eine aktive Kühlung entschieden habe).

Das oben dargestellte System ist an vier 420W-Solarmodule angeschlossen und produziert seit Juni 2024 zuverlässig Strom in meinem Elternhaus. Viel Spass beim Nachbauen!

Michael

**Comment by schlimmchen** [comment]
Hübsch! Erinnert mich an meinen eigenen Aufbau :wink:

![image](https://github.com/user-attachments/assets/6f8ea6b4-ce76-4ad9-b7c8-e3489fa73893)

**Reply by Manos1966** [reply]
Pre-charge? 🤩 
wie gross ist der Widerstand?

![Precharge](https://github.com/user-attachments/assets/48eb94ee-3c95-4b3c-b028-eeeffacdf211)

**Reply by spcqike** [reply]
3Ohm 100W.

**Reply by gruenteefan** [reply]
> Pre-charge? 🤩 wie gross ist der Widerstand?
> 
> ![Precharge](https://private-user-images.githubusercontent.com/129660360/380350243-48eb94ee-3c95-4b3c-b028-eeeffacdf211.JPG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDAxNDc0MDMsIm5iZiI6MTc0MDE0NzEwMywicGF0aCI6Ii8xMjk2NjAzNjAvMzgwMzUwMjQzLTQ4ZWI5NGVlLTNjOTUtNGIzYy1iMDI4LWVlZWZmYWNkZjIxMS5KUEc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMjIxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDIyMVQxNDExNDNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00NTFhNDcyOTdkYTYwMGQwYjYwNDZjMjdiNzc2MDMxMTViNTU4ODhkNTBjMGFmOWQ3ZGIxZGExYzEwY2RkNGY4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.IXR-yAKpF-EhZkOqyiVjpRsYB02rLzmV9qCimPRF17w)

Wozu ist das genau gut?

**Reply by Manos1966** [reply]
> Wozu ist das genau gut?

https://github.com/hoylabs/OpenDTU-OnBattery/wiki/Impact-of-high-start-currents-to-the-inverter

**Comment by gruenteefan** [comment]
Ich habe einen ähnlichen Aufbau realisiert. Habt ihr einen Überspannungs-Schutz für die PV Module eingebaut? Welche Auslegung bräuchte ich hier je String (2x 420W in Reihe)?

**Reply by schlimmchen** [reply]
Meinst du einen Blitzschutz? Worauf muss der denn ausgelegt sein? Blitzschlag funktioniert recht gleich überall auf der Welt...
